docs/Users/Tutorials/External_Bears.rst

```diff
@@ -0,0 +1,511 @@
+External Bears
+==============
+
+Welcome. This tutorial hopes to teach you how to use the
+``@external_bear_wrap`` decorator in order to write Bears in languages other
+than python.
+
+.. note::
+
+  This tutorial assumes that you already know the basics. If you are new please
+  refer to
+  :doc:`this link<Writing_Bears>`.
```

Comment:
"this link" doesn't sound good. Just use "refer to the [Writing Bears](#link to the section) section below" maybe.